### ISSUE TITLE
Stylelint output unstable on npm scripts chaining - requires two passes

### ISSUE THREAD

**Issue by avrelaun** [stylelint/stylelint#3857]
We have a package.json script at work which formats all our codebase: ```prettier --write && eslint --fix && stylelint --fix```. When ran on our codebase, stylelint fails to report and fix some indentations errors.

If we run ```prettier --write && eslint --fix```, then ```stylelint --fix``` stylelint catches the formatting problems and fix thems.

Same thing happens if we run stylelint twice on our fix script, it works as expected by catching and fixing problems: ```prettier -- write && eslint --fix && stylelint --fix && stylelint --fix```

> Which rule, if any, is the bug related to?

indentation (configured with tabs)

> What CSS is needed to reproduce the bug?
> What stylelint configuration is needed to reproduce the bug?

A minimal environment is provided here with everything needed: https://github.com/avrelaun/prettier-stylelint-conflict

> Which version of stylelint are you using?

9.9.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI, example on code provided

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Indentation should should be called

> What actually happened (e.g. what warnings or errors did you get)?

"The following warnings were flagged on yarn run lint:"

```
src/styles/_fonts.scss
   9:2  ⚠  Expected indentation of 2 tabs   indentation
  26:2  ⚠  Expected indentation of 2 tabs   indentation
  42:2  ⚠  Expected indentation of 2 tabs   indentation
  59:2  ⚠  Expected indentation of 2 tabs   indentation
  76:2  ⚠  Expected indentation of 2 tabs   indentation
  93:2  ⚠  Expected indentation of 2 tabs   indentation
 110:2  ⚠  Expected indentation of 2 tabs   indentation

src/styles/_functions.scss
 10:2  ⚠  Expected indentation of 1 tab   indentation
 14:2  ⚠  Expected indentation of 1 tab   indentation
 18:2  ⚠  Expected indentation of 1 tab   indentation

src/styles/_mixins.scss
 220:2  ⚠  Expected indentation of 1 tab    indentation
 226:2  ⚠  Expected indentation of 1 tab    indentation
 232:2  ⚠  Expected indentation of 1 tab    indentation
 308:2  ⚠  Expected indentation of 2 tabs   indentation
 313:2  ⚠  Expected indentation of 1 tab    indentation

src/components/bk-loading.vue
 170:2  ⚠  Expected indentation of 4 tabs   indentation
 181:2  ⚠  Expected indentation of 4 tabs   indentation
 191:2  ⚠  Expected indentation of 5 tabs   indentation
 199:2  ⚠  Expected indentation of 5 tabs   indentation
 210:2  ⚠  Expected indentation of 5 tabs   indentation
 218:2  ⚠  Expected indentation of 5 tabs   indentation

static/init-loader.css
 147:2  ⚠  Expected indentation of 2 tabs   indentation
 158:2  ⚠  Expected indentation of 2 tabs   indentation
 182:2  ⚠  Expected indentation of 3 tabs   indentation
 190:2  ⚠  Expected indentation of 3 tabs   indentation
 201:2  ⚠  Expected indentation of 3 tabs   indentation
 209:2  ⚠  Expected indentation of 3 tabs   indentation

```

**Comment by hudochenkov** [comment]
Thanks for using the template.

I think [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier) will fix your problems. Currently, you're using conflicting rules, which mentioned config will disable.

**Comment by jeddy3** [comment]
Closing for triage.
